content/en/tracing/services/service_page.md

```diff
@@ -98,6 +98,8 @@ Below, there’s a list of [resources][11] associated with your service. Sort th
 
 {{< img src="tracing/visualization/service/resources_tab.jpg" alt="Resources"  style="width:100%;">}}
 
+Click on a resource to view a summary panel with the resource’s out-of-the-box graphs (Requests, Errors, Latency), the Resource Dependency Map, and Span Summary Table. Use keyboard navigation keys to toggle between resources on the resource list to compare resources in a service, and quickly identify issues without context switching. Click “Open Full Page” or use cmd-click or cntrl-click to view the full resource page in a new tab.
```

Comment:
```suggestion
Click on a resource to open a side panel that displays the resource's out-of-the-box graphs (requests by version, errors by version, and latency), a resource dependency map, and a span summary table. Use keyboard navigation keys to toggle between resources on the **Resources** list and compare resources in a service. To view the full resource page, click **Open Full Page**.
```

Reply:
"(requests by version, errors by version, and latency)" - other graphs can be shown through the toggle, can we keep it broad to just "(requests, errors, and latency)"?